tutorials/conventions.ipynb

```diff
@@ -15,14 +15,24 @@
    "source": [
```

Comment:
<p>I suggest using the link to the section, i.e. append #porepy-style to the link. Perhaps we should also format the link, so that it displays as "here"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done